samples/subsys/usb/uac2_implicit_feedback/src/feedback_dummy.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include "feedback.h"
+
+#warning "No target specific feedback code, overruns/underruns will occur"
+
+struct feedback_ctx *feedback_init(void)
+{
+	return NULL;
+}
+
+void feedback_process(struct feedback_ctx *ctx)
```

Comment:
With should normally be executed in these functions? Can some comments be added to guide customers?

Reply:
Read the comment in feedback.h:
https://github.com/zephyrproject-rtos/zephyr/blob/602054402313f885c2a722616ceee87c735668b6/samples/subsys/usb/uac2_implicit_feedback/src/feedback.h#L22-L33

The comment refers to what has to be reported. How this is achieved is entirely up to the implementer.

I only know how to achieve it using DPPI subsystem and TIMER on nRF53. I hope that eventually someone will come up with platform independent solution, but this is not a trivial task at all and I think it would require some sort of interrupt/event timestamping API that is simply available yet (and some clever filtering because the control goal is to have I2S FRAMESTART and USB SOF happen at the same time, where obviously one interrupt handler will execute before the other and the two will keep "swapping" with each other which one executes first).